frontend/src/components/apps/useAppFunctionsColumns.tsx

```
@@ -1,14 +1,14 @@
 "use client";
 
 import { FunctionDetail } from "@/components/apps/function-detail";
-import { type AppFunction } from "@/lib/types/appfunction";
+import { type Function } from "@/lib/types/function";
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)